.editorconfig

```diff
@@ -0,0 +1,36 @@
+# This is the top-most EditorConfig file.
+root = true
+
+# For all files.
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = false
```

Comment:
Pretty sure this is `true` for our project?

Reply:
Fixed